src/util/Utils.cpp

```diff
@@ -215,6 +217,20 @@ std::string Utils::getFileName(std::string filePath) {
     return filename;
 }
 
+/*
+ * Get the current users(calle of the program) home directory from the $HOME environment variable,
```

Comment:
Change "Get the current users(calle of the program)" to "Get the current user's (caller of the program)"

Reply:
Fixed in 532a9a7 commit